src/task/JoinAndReceiveIndexTask.ts

```diff
@@ -111,9 +112,19 @@ export default class JoinAndReceiveIndexTask extends BaseTask {
       this.context.previousSdpOffer = null;
       this.context.serverSupportsCompression = false;
 
-      this.context.signalingClient.join(
-        new SignalingClientJoin(this.context.meetingSessionConfiguration.applicationMetadata)
+      const join = new SignalingClientJoin(
+        this.context.meetingSessionConfiguration.applicationMetadata
       );
+      if (this.context.videoDownlinkBandwidthPolicy.wantsServerSideNetworkAdaption !== undefined) {
+        switch (this.context.videoDownlinkBandwidthPolicy.wantsServerSideNetworkAdaption()) {
```

Comment:
Will the other case here is serverSideNetworkAdaptation? If that is the case, should we just make one property in SignalingJoin constructor for `wantsServerSideNetworkAdaption` and pass through the properties there.
A bit confuse why we need to break out these enum values into 2 different boolean props.

Reply:
Ya...I've been meaning to do that (originally was using bools on the config as well), just a bit awkward because I need to deprecate the previous flags (should be fine to do before release :) ).